### DISCUSSION TITLE
Does it support Chia-1.2.0  for new pool compatable plot?

### DISCUSSION THREAD

**Discussion by feiyingCQ** [swar/Swar-Chia-Plot-Manager#1232] [Q&A]
Any configuration need to be changed  if I want to use latest chia-1.2.0 to plot new portable plots?

**Comment by NomisSplat** [comment]
Not yet. I am looking to see if I can alter the code to hack it to hard code the option in right now (without using the config.yaml file)...

**Reply by feiyingCQ** [reply]
Thanks.  config.yaml is good stuff.  waiting for Swar-Chia-Plot-Manager creator to implement the new feature.  
Can't wait to mine in a pool......  180T plots SOLO one month, got nothing till now

**Reply by bryantcj89** [reply]
Creator please do something PLS!

**Reply by MyLabs-LLC** [reply]
> Not yet. I am looking to see if I can alter the code to hack it to hard code the option in right now (without using the config.yaml file)...

it we have to modify the config file to get NFTs to work that is fine, we just all really lover your tool

**Reply by CakeRepository** [reply]
If someone finds where it executes the chia.exe we could likely manipulate it with a pull request I was looking earlier

**Reply by Chilli71** [reply]
Update: I just see it should already work with the offical version.

> If someone finds where it executes the chia.exe we could likely manipulate it with a pull request I was looking earlier

I have never programmed python in my live and not planned to start it, yet.
in plots.py alter line 17 from
`        flags['p'] = pool_public_key`
to
`        flags['c'] = pool_public_key`
and in the config.yaml file specify the pool contract key at the variable 'pool_public_key' instead of specifying the pool_public_key itself.
and test if the solution works on your system before you start ramping up the plots

Cheers
Chilli

**Comment by TomasVelPar** [comment]
if you check this, you could have it ruinning with the new procotol: [https://github.com/swar/Swar-Chia-Plot-Manager/pull/1224](https://github.com/swar/Swar-Chia-Plot-Manager/pull/1224)

**Comment by swar** [comment]
The new pool protocol stuff has been merged.
